content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+
+## Troubleshooting
+
+While the above process has been tested several times and tested succesfully the retention of data, there were a couple of issues that you may run into and are documented below
+
+### Postgres pod will not come up after deleting the pod
+
+In the first test above, after deleting the Postgres pod, the new one would not come up. After looking at the logs of the pod found errors like 
+
+```shell
+find: ‘/var/lib/postgresql/data/pgdata/pg_stat_tmp/global.stat’: Structure needs cleaning
+chown: cannot access '/var/lib/postgresql/data/pgdata/pg_stat_tmp/global.stat': Structure needs cleaning
+```
+This appeared to be more of an OS issue, so I deleted this pod. When the new one was scheduled all was fine.
+
+### Postgres unable to mount to data directory
+
+In some occassions when you shut down the node that Postgres is running on, you may run into a situation where the new pod that gets scheduled on one of the remaining nodes is not able to come up due to volume mount issues. Running `kubectl describe` on the Postgres pod may show an error like the following:
+
+![PostgresDesribe](/images/guides/kots/ha-cluster-postgres-describe.png)
+
+The first thing to check is to ensure you have followed the [configuring ekco] section. Ekco helps with ensuring that any resources that were running on the stopped node are properly deleted and moved to a different node. Without ekco, the claim on the PVC associated to the Postgres pod that was running on the stopped node sometimes doesn't get properly cleared and when the new pod gets scheduled on another node, it is not able to lay a claim to the volume.
+
+### Other Persistent Data Issues
+
+The default configuration of the Kurl installer includes the [rook add-on](https://kurl.sh/docs/add-ons/rook), which creates and manages a [Ceph cluster](https://docs.ceph.com/docs/master/rados/) along with a storage class for provisioning Persistent Volume Claims (PVCs). If you are running into other issues with persisitent storage, following are some helpul commands to check the health of ceph in your cluster. Keep in mind that the screenshots below show a `healthy` cluster that has all nodes up and running. 
+
+All related pods are runnning in the rook-ceph namespace, and a good place to start to troubleshoot any issues. By running `kubectl get pods -n rook-ceph` you should get an output similar to the one below:
+
+![PostgresDesribe](/images/guides/kots/ha-cluster-rook-ceph-pods.png)
+
+One thing to note here is that you will see pods running on all three nodes. Orchestrating all of the data accross all nodes is the `rook-ceph-operator-...` pod. To troubleshoot issues, we'll need to exec into this pod in order to run some commands. For example, to exec into the pod on the screenshot above, I would run `kubectl exec -it rook-ceph-operator-6fbfdccc57-p477z`. As you exec into the pod, you may see a few errors but you can igonre them.
```

Comment:
This seems dangerous `As you exec into the pod, you may see a few errors but you can igonre them.` -- do you have an example of what kind of errors can be ignored? (also typo in ignore)

Reply:
I just removed the sentence :)